README.adoc

```diff
@@ -38,28 +33,24 @@ $ manifest-tool --username * --password * push from-spec config-server/manifest.
 |link:config-server/[steeltoeoss/config-server]
 | Spring Cloud Config Server
 
-|link:dotnet-runtime/[steeltoeoss/dotnet-runtime]
-| Dotnet Runtime
-
 |link:eureka-server/[steeltoeoss/eureka-server]
 | Netflix Eureka Server
 
-|link:hystrix-dashboard/[steeltoeoss/hystrix-dashboard]
-| Netflix Hystrix Dashdboard
-
-|link:mssql/[steeltoeoss/mssql]
-| SQL Server
+|link:spring-boot-admin/[steeltoeoss/spring-boot-admin]
+| Spring Boot Admin
+|link:uaa-server/[steeltoeoss/sample-uaa-server]
```

Comment:
This looks like a mistake to me, can you confirm?
```suggestion
|link:uaa-server/[steeltoeoss/uaa-server]
```

Reply:
Building this one failed on my machine:
```
./build.ps1 uaa-server
docker build -t steeltoeoss/uaa-server-amd64-linux C:\Source\Repos\Dockerfiles\uaa-server
2024/02/26 12:15:35 http2: server: error reading preface from client //./pipe/docker_engine: file has already been closed
[+] Building 49.4s (10/11)                                                                                                                                                                                                            docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                                                                            0.0s
 => => transferring dockerfile: 341B                                                                                                                                                                                                            0.0s
 => [internal] load metadata for docker.io/adoptopenjdk/openjdk11:latest                                                                                                                                                                        1.6s
 => [internal] load .dockerignore                                                                                                                                                                                                               0.0s
 => => transferring context: 2B                                                                                                                                                                                                                 0.0s
 => [source 1/6] FROM docker.io/adoptopenjdk/openjdk11:latest@sha256:54cc1055174eb77b2922e158a71afebc14189c745b609eec9716de6498b55eb2                                                                                                           6.7s
 => => resolve docker.io/adoptopenjdk/openjdk11:latest@sha256:54cc1055174eb77b2922e158a71afebc14189c745b609eec9716de6498b55eb2                                                                                                                  0.0s
 => => sha256:ddb7f46b1c092c689ea85f1c7025a7bdfedf5e7bac0c85ed29a7fefa06a9c1a2 195.86MB / 195.86MB                                                                                                                                              5.3s
 => => sha256:54cc1055174eb77b2922e158a71afebc14189c745b609eec9716de6498b55eb2 1.72kB / 1.72kB                                                                                                                                                  0.0s
 => => sha256:b43c22e4bec8074926bdb718b244e8ff27acab76a0609c9bff3016ab1b9e0078 954B / 954B                                                                                                                                                      0.0s
 => => sha256:887586fa4fe8621b560703441839c60553479cbccea81aa3af162b03b9295d92 4.54kB / 4.54kB                                                                                                                                                  0.0s
 => => sha256:8ee0874247356ecb5ea92128219660506b139dcb6cc45dcab84d98b3c6485061 27.51MB / 27.51MB                                                                                                                                                1.1s
 => => sha256:efd39554e4765346c1f62ee2d86960bbe724e8c9f0ac5b2c75c3781f114f6aa0 16.06MB / 16.06MB                                                                                                                                                1.6s
 => => extracting sha256:8ee0874247356ecb5ea92128219660506b139dcb6cc45dcab84d98b3c6485061                                                                                                                                                       1.0s
 => => extracting sha256:efd39554e4765346c1f62ee2d86960bbe724e8c9f0ac5b2c75c3781f114f6aa0                                                                                                                                                       1.0s
 => => extracting sha256:ddb7f46b1c092c689ea85f1c7025a7bdfedf5e7bac0c85ed29a7fefa06a9c1a2                                                                                                                                                       1.2s
 => [internal] load build context                                                                                                                                                                                                               0.0s
 => => transferring context: 8.45kB                                                                                                                                                                                                             0.0s
 => [source 2/6] RUN apt-get update && apt-get install -y git                                                                                                                                                                                  25.8s
 => [source 3/6] RUN git clone -b v75.0.0 https://github.com/cloudfoundry/uaa.git                                                                                                                                                               5.6s
 => [source 4/6] COPY uaa.yml /uaa/uaa/src/main/resources/uaa.yml                                                                                                                                                                               0.0s
 => [source 5/6] WORKDIR /uaa                                                                                                                                                                                                                   0.0s
 => ERROR [source 6/6] RUN ./gradlew assemble                                                                                                                                                                                                   9.7s
------
 > [source 6/6] RUN ./gradlew assemble:
0.436 Downloading https://services.gradle.org/distributions/gradle-6.3-bin.zip
1.361 .................................................................................................
4.073
4.073 Welcome to Gradle 6.3!
4.073
4.073 Here are the highlights of this release:
4.073  - Java 14 support
4.076  - Improved error messages for unexpected failures
4.076
4.076 For more details see https://docs.gradle.org/6.3/release-notes.html
4.076
4.077 Starting a Gradle Daemon (subsequent builds will be faster)
9.155
9.155 FAILURE: Build failed with an exception.
9.155
9.156 * What went wrong:
9.156 A problem occurred configuring root project 'cloudfoundry-identity-parent'.
9.156 > Could not resolve all artifacts for configuration ':classpath'.
9.156    > Could not resolve io.spring.gradle:propdeps-plugin:0.0.10.RELEASE.
9.156      Required by:
9.156          project :
9.156       > Could not resolve io.spring.gradle:propdeps-plugin:0.0.10.RELEASE.
9.156          > Could not get resource 'https://repo.spring.io/plugins-release/io/spring/gradle/propdeps-plugin/0.0.10.RELEASE/propdeps-plugin-0.0.10.RELEASE.pom'.
9.156             > Could not GET 'https://repo.spring.io/plugins-release/io/spring/gradle/propdeps-plugin/0.0.10.RELEASE/propdeps-plugin-0.0.10.RELEASE.pom'. Received status code 401 from server:
9.157
9.157 * Try:
9.157 Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
9.157
9.157 * Get more help at https://help.gradle.org
9.157
9.157 BUILD FAILED in 8s
------
Dockerfile:6
--------------------
   4 |     COPY uaa.yml /uaa/uaa/src/main/resources/uaa.yml
   5 |     WORKDIR /uaa
   6 | >>> RUN ./gradlew assemble
   7 |
   8 |     FROM source as run
--------------------
ERROR: failed to solve: process "/bin/sh -c ./gradlew assemble" did not complete successfully: exit code: 1

View build details: docker-desktop://dashboard/build/default/default/6vpkfogtsh22v414576jxrxop
```

Probably not relevant to this PR, as its description states it is out of scope. Just thought I should mention this.

Reply:
Definitely out of scope from this PR. The last time I tried to build UAA was years ago and something similar happened where a pinned version couldn't be built anymore. The UAA image is something that could be revisited during the Steeltoe Security review